src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -149,8 +148,8 @@ public String getLabel() {
     }
 
     @Override public StepExecution start(StepContext context) throws Exception {
-        if (this.label != null) {
-            context.get(FlowNode.class).addAction(new LabelAction(StringUtils.left(label, MAX_LABEL_LENGTH)));
+        if (label != null) {
+            context.get(FlowNode.class).addAction(new LabelAction(label.length() > MAX_LABEL_LENGTH ? label.substring(0, MAX_LABEL_LENGTH) : label));
```

Comment:
Dubious logic from #93; IMO better for any truncation to be done in display plugins, not here.